src/tests/playwright/pages/perf-gain.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script defer type="module">
+      import record from './processors/recorder/recorder-main.js';
+      import { beCloseTo, test, evaluate } from './util/audit.js';
+
+      test((async () => {
+        const ctx = new AudioContext({sampleRate: 48000});
+        const src = new ConstantSourceNode(ctx);
+        const gs = Array(100).fill().map(() => new GainNode(ctx));
```

Comment:
What does `gs` mean?

Reply:
I would spell out everything here. For example:
`ctx` - `context`
`src` - `constantSource`
`gs` - `gainNodes`